rust/src/xenctrl.rs

```diff
@@ -196,7 +196,7 @@ impl Xenctrl {
           n => {
             let info_res = &info[0..n as usize];
             for i in 0..n {
-              let info_i = info[i as usize];
+              let info_i: DomainInfo = info[i as usize];
```

Comment:
Why?

Reply:
i was testing things, i already removed it